core/src/main/java/com/snowgears/shop/Shop.java

```diff
@@ -160,22 +165,23 @@ public void onLoad(){
         hookWorldGuard = config.getBoolean("hookWorldGuard");
         // Check if WorldGuard exists
         // Note: If WorldGuard exists we will check to verify a user can build in the region
-        if (getServer().getPluginManager().getPlugin("WorldGuard") != null) {
-            this.getLogger().notice("WorldGuard detected, Shop will respect `passthrough`, `build`, and `chest-access` region flags during shop creation!");
-            // Store for later
-            this.worldGuardExists = true;
+        // if (getServer().getPluginManager().getPlugin("WorldGuard") != null) {
```

Comment:
We perform the check to see if the WorldGuard plugin is available to make sure that we don't attempt to register flags when WorldGuard is not present. Additionally, even if the "hookWorldGuard" flag is not set, we still want to follow WorldGuard restrictions always since it is a protection plugin. The  "hookWorldGuard" flag just controls if we should enforce the "allow-shop" flag.

We still want to log that we will respect `passthrough`, `build`, and `chest-access` so that it is clear to the user what is occuring.

Reply:
We should also remove code instead of commenting it out, leaving commented code in the codebase can often lead to confusion later down the road as to why it was disabled.